### ISSUE TITLE
hard to train yolo v1  contain loss

### ISSUE THREAD

**Issue by Tshzzz** [pjreddie/darknet#774]
I try to implement yolov1 by pytorch. but I find contain loss is hard to train.... 
I try to make the contain value regress the iou value 
obj_contain_loss = F.mse_loss(obj_contain_pred[:,4],max_iou,size_average=False)     

and regress the 1 value:
obj_contain_loss = F.mse_loss(obj_contain_pred,obj_target_contain,size_average=False)     

I don't should I write right code , Am I right?
